packages/tangle/images/TestBookerMarkerMappings.md

```diff
@@ -0,0 +1,230 @@
+    - PastMarker.BranchID = Main
+    - Payload.BranchID = Main
+
+![](https://imgur.com/GIJjo0b.png)
+
+### Transaction 5 (create Marker): 
+
+- message 5
+    - PastMarker.BranchID = Main
+    - Payload.BranchID = Main
+
+![](https://imgur.com/YMGAbiF.png)
+
+### Transaction 6 (create Marker): 
+
+- message 6
+    - PastMarker.BranchID = [A], [.] indicates inheritance
+    - Payload.BranchID = Main, since no double spend
+
+![](https://imgur.com/ZtSYkvq.png)
+
+### Transaction 7 (create Marker): 
+
+- message 7
+    - PastMarker.BranchID = [C], [.] indicates indirect mapping of the marker to the Branch
+    - Payload.BranchID = Main, since no double spend
+
+![](https://imgur.com/eqmTCnY.png)
+
+### Transaction 8 (create Marker): 
+
+- message 8
+    - PastMarker.BranchID = [A], [.] indicates indirect mapping of the marker to the Branch, 
+    - Payload.BranchID = Main, since no double spend
+
+![](https://imgur.com/gbLlGgd.png)
+
+### Transaction 9 (create Marker, double spend): 
+
+- message 9
+    - PastMarker.BranchID = [C], [.] indicates indirect mapping of the marker to the Branch
+    - Payload.BranchID = Main, since no double spend
+
+![](https://imgur.com/i6ZFvnF.png)
+
+### Transaction 10 (create Marker, double spend): 
+
+- message 5
+    - create Branch D
+    - change: PastMarker.BranchID = D
+    - change: Payload.BranchID = D
+- message 10
+    - double spends message 5
+    - create Branch E
+    - PastMarker.BranchID = B+E
+    - Payload.BranchID = E
+
+![](https://imgur.com/lImMAQY.png)
+
+### Transaction 11 (create Marker): 
+
+- Message 11: 
+    - PastMarker.BranchID = A+C
+    - Payload.BranchID = Main
+
+![](https://imgur.com/WT0qEEH.png)
+
+### Transaction 12: 
+
+- Message 12: 
+    - no marker !
+    - PastMarker.BranchID = (1,3), (3,3)
+    - Payload.BranchID = Main
+    - MessageMetadata.BranchID = A+C
+
+![](https://imgur.com/yTk30Zq.png)
+
+### Transaction 13 (create Marker): 
+
+- Message 13: 
+    - PastMarker.BranchID = A+C+E
+    - Payload.BranchID = Main
+
+![](https://imgur.com/eZ6DPr4.png)
+
+### Transaction 14: 
+
+- Message 14: 
+    - no marker !
+    - PastMarker.BranchID = (1,3), (3,3)
+    - Payload.BranchID = Main
+    - MessageMetadata.BranchID = A+C
+
+![](https://imgur.com/W8p0Ahw.png)
+
+### Transaction 15 (create Marker): 
+
+- Message 15: 
+    - PastMarker.BranchID = [A+C], [.] indicates inheritance
+    - Payload.BranchID = Main
+
+![](https://imgur.com/RBbzhwp.png)
+
+### Transaction 16 (create Marker): 
+
+- Message 16: 
+    - PastMarker.BranchID = [A+C], [.] indicates inheritance
+    - Payload.BranchID = Main
+
+![](https://imgur.com/K6BbedV.png)
+
+### Transaction 17 (create Marker, double spend): 
+
+- Message 17: 
+    - double spends message 6
+    - create Branch G
+    - PastMarker.BranchID = G
+    - Payload.BranchID = G
+- Message 6:
+    - create Branch F
+    - change PastMarker.BranchID = F
+- Message 8: 
+    - change PastMarker.BranchID = F
```

Comment:
Maybe it should be mentioned that no action/update of mapping is necessary here because of the inheritance it is implicit. Same goes for message 15 and 16 in this case.